RFCs/Template.md

```diff
@@ -0,0 +1,34 @@
+# RFC-#: Title
+
+**To be Reviewed By:** 
+
+**Authors:**
+
+**Status:** **Active** | Accepted | Dropped | Superseded
```

Comment:
Is there a "Draft" state too? See earlier comment.

Reply:
I don't think it buys us much. When there is a PR all can comment, hence Active.